client/src/components/statistics/Statistics.js

```diff
@@ -0,0 +1,40 @@
+import SeriesChart from "./SeriesChart";
+import Pagination from "../pagination/Pagination";
+import seriesService from "../../services/series.service";
+import { useEffect, useState } from 'react';
+
+const Statistics = () => {
+  const [series, setSeries] = useState([]);
+  const [page, setPage] = useState(1);
+  const [totalCount, setTotalCount] = useState(0);
+  const [pageSize] = useState(10);
+
+  useEffect(() => {
+    const fetchData = async() => {
+      const response = await seriesService.getMostPopularSeries(page);
+      setSeries(Array.from(response.data));
+      setTotalCount(response.totalAmount)
+    }
+    fetchData();
+  }, [page]);
+
+  const onPageChange = (newPage) => {
+    setPage(newPage);
+  }
+
+  return (
+    <div style={{ display: "flex", flexDirection: "column", alignItems: 'center' }}>
+      <h3>Series Statistics</h3>
+      <SeriesChart series={series} dataKey="popularity" />
+      <SeriesChart series={series} dataKey="vote_average" />
+      <Pagination
```

Comment:
Can you try to move Pagination component a bit higher in CSS? It's quite far down from the charts